pkgs/by-name/sh/shoko/package.nix

```
@@ -0,0 +1,54 @@
+{
+  buildDotnetModule,
+  fetchFromGitHub,
+  dotnet-sdk_8,
+  dotnet-aspnetcore_8,
+  nixosTests,
+  lib,
+  mediainfo,
+  rhash,
+  nix-update-script
```

`nixfmt-check` is complaining again:
```suggestion
  nix-update-script,
```

In the future you could try running `nix fmt` inside the `nixpkgs` folder to format every file. At first it can takes a few minutes but after that its ~300ms.